about.html

```diff
@@ -402,7 +402,7 @@ <h2>Get a Quote</h2>
                       </div>
                       <ul class="mb-40">
                         <li class="number">
-                          <a href="#"> +1 206 555 0101</a>
+                          <a href="#">06 27 56 01 59</a>
```

Comment:
Même chose, préfixe international : 

```suggestion
                          <a href="#">+33 6 27 56 01 59</a>
```